code/game/objects/items/grenades/flashbang.dm

```diff
@@ -26,11 +26,15 @@
 
 //Flash
 	if(M.flash_act(affect_silicon = 1))
-		M.confused += (max(20/max(1,distance), 6))
+		M.Paralyze(max(20/max(1,distance), 5))
+		M.Knockdown(max(200/max(1,distance), 60))
 //Bang
 	if(!distance || loc == M || loc == M.loc)	//Stop allahu akbarring rooms with this.
-		var/protection = max(1, M.get_ear_protection())
-		M.adjustEarDamage(15/protection, 30/protection)
+		M.Paralyze(20)
```

Comment:
Wont this paralyze people with ear protection?

Reply:
It does not, hearing protection is accounted for. The distance check was added to prevent people from dragging flashbangs behind them3